ansible/roles/prerequisites/tasks/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: Install podman and podman-compose
+  become: yes
```

Comment:
Tem algum problema em botar esse become no playbook todo?

Reply:
Eu acredito que isso vai forçar os containers a rodarem sempre como root.